functions/Set-DbaAgentAlert.ps1

```diff
@@ -132,7 +132,7 @@ function Set-DbaAgentAlert {
             #region alert options
             # Settings the options for the alert
             if ($NewName) {
-                if ($Pscmdlet.ShouldProcess($server, "Setting alert name to $NewName for $Alert")) {
+                if ($Pscmdlet.ShouldProcess($server, "Setting alert name to $NewName for $a")) {
```

Comment:
Why have you changed that? $a is only the last alert from the foreach loop. Shouldn't it be `$currentAlert`?